weather.go

```diff
@@ -25,26 +26,23 @@ type Weather struct {
 		Pressure  int     `json:"pressure"`
 		Humidity  int     `json:"humidity"`
 	} `json:"main"`
+	Name string `json:"name"`
 }
 
 type CityUnknown struct {
 	Cod     string `json:"cod"`
 	Message string `json:"message"`
 }
 
-type CliResponse struct {
+type Conditions struct {
+	City    string
 	OneWord string  `json:"oneword"`
 	Celcius float64 `json:"celcius"`
 }
 
-type Response struct {
-	OneWord string  `json:"oneword"`
-	Celcius float64 `json:"celcius"`
-}
-
-func CliOutput(token, location string) (output string) {
+func CliOutput(token string, location *strings.Builder) (output string) {
```

Comment:
If what we _want_ is a string, it's more straightforward to pass a string, isn't it? A `strings.Builder` is useful for _constructing_ strings, as the name suggests, but it's an awkward thing for conveying them over long distances.

Reply:
fixed